client/src/components/TextInputComponent/TextInput.css

```diff
@@ -0,0 +1,6 @@
+input {
```

Comment:
Hey, if you use the base html tag here, it'll class with the styles anywhere else that is using the base tag. Instead, can you use a class?